docs/rest-api.yml

```diff
@@ -214,7 +214,7 @@ paths:
           type: string
       responses:
         '200':
-          description: Sets a cookie granting the requesting admin access to the user's server
+          description: Sets a cookie granting admin access to the user's notebook server
```

Comment:
Same here - "the requesting admin" is important information. Perhaps "requesting administrator"?